core/include/kernel/tee_ta_manager.h

```diff
@@ -37,6 +37,9 @@
 #include "tee_ta_manager_unpg.h"
 #include "utee_types.h"
 
+/* Magic TEE identity pointer: set when teecore requests a TA close */
```

Comment:
Write `TEE_Identity` since it's the real type.

If I understand well, you pass KERN_IDENTITY when the kernel requests an operation. You pass NULL when the operation is initiated by the non-secure side (client app). And you pass an actual TEE_Identity when the operation is initiated by another TA.
If so, I would suggest to use a custom define also for the second case instead of NULL. Such as:
```
#define NSAPP_IDENTITY (NULL)    /* Operation is initiated by a client (non-secure) app */
```